src/main/java/com/lina/airline/utils/ApplicationConstantsUtils.java

```diff
@@ -0,0 +1,7 @@
+package com.lina.airline.utils;
+
+public final class ApplicationConstantsUtils {
+    public  static final  String JWT_SECRET_KEY="JWT_SECRET";
```

Comment:
This properties should takes from the application properties file. This value may changes based on environment.